common/eq_packet_structs.h

```diff
@@ -4165,9 +4164,14 @@ struct RaidAddMember_Struct {
 /*139*/	uint8 flags[5]; //no idea if these are needed...
 };
 
+struct RaidNote_Struct {
+/*000*/ RaidGeneral_Struct general;
+/*140*/ std::string note;
+};
+
 struct RaidMOTD_Struct {
 /*000*/ RaidGeneral_Struct general; // leader_name and action only used
-/*136*/ char motd[0]; // max size is 1024, but reply is variable
+/*140*/ std::string motd; // max size is 1024, but reply is variable
```

Comment:
After looking at this closer this should be safe even if the packet is queued as long as the struct containing the string is being dynamically allocated as the packet's buffer,

Reply:
Looking at this again, I think these should be kept as `char[]` to avoid possible undefined behavior with memcpy and destruction